samples/subsys/audio/sof/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_METEORLAKE=y
+CONFIG_INTEL_DMIC=y
+CONFIG_INTEL_SSP=y
+CONFIG_INTEL_ALH=y
+CONFIG_LP_MEMORY_BANKS=1
+CONFIG_HP_MEMORY_BANKS=22
```

Comment:
These aren't Zephyr kconfigs, and SOF really shouldn't be touching the SRAM power control anymore.  If this is actually needed, we probably have some code in SOF that needs to be disabled?  Or maybe it's just some vestigial stuff copied from elsewhere?

Reply:
@andyross Good catch. Support for getting these via dt was added recently -> "soc/intel_adsp: fetch HPSRAM bank count from devicetree" FYI @aborisovich

Reply:
Unfortunately this is not applicable to ACE platforms.
We have a 2816K SRAM memory and it does divide to this fixed SRAM_BANK_SIZE at all.
In fact we have a 8 MB virtual addressing to access HPSRAM and dividing total memory by this number does not give you EBBs count. We need to figure out something alternative here,

Reply:
Done.